CONTRIBUTOR.md

```diff
@@ -0,0 +1,48 @@
+# Contributing to Protobom
+
+We welcome contributions to the protobom project! By participating in this project, you agree to abide by the [Code of Conduct](code-of-conduct.md).
+
+## Getting Started
+
+To contribute to the protobom repository, follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your contribution:
+   ```
+   git checkout -b feature/new-feature
+   ```
+3. Make your desired changes to the codebase.
+4. Commit your changes with a descriptive commit message.
+5. Push your branch to your forked repository:
+   ```
+   git push origin feature/new-feature
+   ```
+6. Open a pull request against the `main` branch of the protobom repository.
+
+
+## Development Guide
+
+For detailed information on setting up your development environment and contributing to the protobom repository, please refer to the [Development Guide](/docs/development.md).
+
+## Code Style
+
+We strive to maintain a consistent code style throughout the project. When contributing to the Go library, please ensure your code follows the following guidelines:
+
+- Use meaningful variable and function names following the camel case convention (`myVariable`, `myFunction`).
+- Write clear and concise comments to describe your code's purpose and functionality.
+- Adhere to the official Go formatting guidelines by running `gofmt` or using an editor that automatically formats your code.
+- Follow the best practices and idiomatic style described in [Effective Go](https://golang.org/doc/effective_go.html).
+
+## Testing
+
+We highly encourage writing tests for new features and bug fixes. This ensures the stability and reliability of the codebase. Reference [README.md](test/conformance/README.md) for conformance testing. 
```

Comment:
Conformance tests are separate from unit/integration tests. Perhaps we should go into what each does (happy to help adding some blurb here)

Reply:
Should there be test/unit_testing/README.md added to the repo? All I see under testing is conformance.

Reply:
Not really, I think the regular tests should be what regular go contributors expect. The conformance tests are about ensuring the integrity of the data, which is why we keep a separate README. Just mentioning in the contributing.md file that we strive to keep our test coverage high so please add tests to any functions you introduce should be enough.  That and a link to the conformance testing.